<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head><title>TestNG Report</title><style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}</style></head><body><table><tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr><tr><th colspan="7">Rohith test Suite</th></tr><tr><td><a href="#t0">Rohith&apos;s test</a></td><td class="num">1</td><td class="num attn">1</td><td class="num attn">18</td><td class="num">864,762</td><td></td><td></td></tr></table><table><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">Rohith test Suite</th></tr></tbody><tbody id="t0"><tr><th colspan="4">Rohith&apos;s test &#8212; failed</th></tr><tr class="failedeven"><td rowspan="18">javapracticeone.PracticeMethods</td><td><a href="#m0">checkPrimeNumber</a></td><td rowspan="1">1499007651195</td><td rowspan="1">21</td></tr><tr class="failedeven"><td><a href="#m1">closeFrames</a></td><td rowspan="1">1499008169845</td><td rowspan="1">32856</td></tr><tr class="failedeven"><td><a href="#m2">countDropdown</a></td><td rowspan="1">1499007885075</td><td rowspan="1">33330</td></tr><tr class="failedeven"><td><a href="#m3">countFrames</a></td><td rowspan="1">1499008118562</td><td rowspan="1">51232</td></tr><tr class="failedeven"><td><a href="#m4">fibonacciSeries</a></td><td rowspan="1">1499007657381</td><td rowspan="1">25</td></tr><tr class="failedeven"><td><a href="#m5">handlingAlerts</a></td><td rowspan="1">1499007987426</td><td rowspan="1">34163</td></tr><tr class="failedeven"><td><a href="#m6">handlingFrame</a></td><td rowspan="1">1499008021803</td><td rowspan="1">36200</td></tr><tr class="failedeven"><td><a href="#m7">handlingPrompts</a></td><td rowspan="1">1499008058046</td><td rowspan="1">34248</td></tr><tr class="failedeven"><td><a href="#m8">handlingWindows</a></td><td rowspan="1">1499008092319</td><td rowspan="1">26226</td></tr><tr class="failedeven"><td><a href="#m9">printAllLinks</a></td><td rowspan="1">1499007918479</td><td rowspan="1">22918</td></tr><tr class="failedeven"><td><a href="#m10">printEvenNumber</a></td><td rowspan="1">1499007651282</td><td rowspan="1">20</td></tr><tr class="failedeven"><td><a href="#m11">printLinks</a></td><td rowspan="1">1499007762922</td><td rowspan="1">122093</td></tr><tr class="failedeven"><td><a href="#m12">readContent</a></td><td rowspan="1">1499008202706</td><td rowspan="1">23001</td></tr><tr class="failedeven"><td><a href="#m13">starPattern</a></td><td rowspan="1">1499008225712</td><td rowspan="1">7</td></tr><tr class="failedeven"><td><a href="#m14">useAllMethods</a></td><td rowspan="1">1499007941476</td><td rowspan="1">45943</td></tr><tr class="failedeven"><td><a href="#m15">useTwitterUsingObjectRepo</a></td><td rowspan="1">1499007657410</td><td rowspan="1">105504</td></tr><tr class="failedeven"><td><a href="#m16">usingLoops</a></td><td rowspan="1">1499007651304</td><td rowspan="1">6074</td></tr><tr class="failedeven"><td><a href="#m17">usingWrapperMethods</a></td><td rowspan="1">1499008225722</td><td rowspan="1">139029</td></tr><tr><th colspan="4">Rohith&apos;s test &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="1">javapracticeone.PracticeMethods</td><td><a href="#m18">dataUsingDataProvider</a></td><td rowspan="1">1499008364825</td><td rowspan="1">0</td></tr><tr><th colspan="4">Rohith&apos;s test &#8212; passed</th></tr><tr class="passedeven"><td rowspan="1">javapracticeone.PracticeMethods</td><td><a href="#m19">usingDropDown</a></td><td rowspan="1">1499007500479</td><td rowspan="1">150712</td></tr></tbody></table><h2>Rohith&apos;s test</h2><h3 id="m0">javapracticeone.PracticeMethods#checkPrimeNumber</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.checkPrimeNumber(PracticeMethods.java:153)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m1">javapracticeone.PracticeMethods#closeFrames</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.closeFrames(PracticeMethods.java:841)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m2">javapracticeone.PracticeMethods#countDropdown</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.countDropdown(PracticeMethods.java:411)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m3">javapracticeone.PracticeMethods#countFrames</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.countFrames(PracticeMethods.java:787)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m4">javapracticeone.PracticeMethods#fibonacciSeries</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.fibonacciSeries(PracticeMethods.java:251)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m5">javapracticeone.PracticeMethods#handlingAlerts</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.handlingAlerts(PracticeMethods.java:537)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m6">javapracticeone.PracticeMethods#handlingFrame</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.handlingFrame(PracticeMethods.java:588)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m7">javapracticeone.PracticeMethods#handlingPrompts</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.handlingPrompts(PracticeMethods.java:658)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m8">javapracticeone.PracticeMethods#handlingWindows</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.handlingWindows(PracticeMethods.java:741)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m9">javapracticeone.PracticeMethods#printAllLinks</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.printAllLinks(PracticeMethods.java:449)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m10">javapracticeone.PracticeMethods#printEvenNumber</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.printEvenNumber(PracticeMethods.java:178)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m11">javapracticeone.PracticeMethods#printLinks</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.printLinks(PracticeMethods.java:376)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m12">javapracticeone.PracticeMethods#readContent</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.readContent(PracticeMethods.java:883)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m13">javapracticeone.PracticeMethods#starPattern</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.starPattern(PracticeMethods.java:908)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m14">javapracticeone.PracticeMethods#useAllMethods</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.useAllMethods(PracticeMethods.java:496)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m15">javapracticeone.PracticeMethods#useTwitterUsingObjectRepo</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.useTwitterUsingObjectRepo(PracticeMethods.java:317)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 15 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m16">javapracticeone.PracticeMethods#usingLoops</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.usingLoops(PracticeMethods.java:224)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m17">javapracticeone.PracticeMethods#usingWrapperMethods</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:505)
	at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1441)
	at org.apache.poi.POIXMLDocument.write(POIXMLDocument.java:201)
	at javapracticeone.PracticeMethods.usingWrapperMethods(PracticeMethods.java:941)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /docProps/app.xml fail to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@1fe4a43
	at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:496)
	... 16 more
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m18">javapracticeone.PracticeMethods#dataUsingDataProvider</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">org.testng.TestNGException: 
Method public void javapracticeone.PracticeMethods.dataUsingDataProvider(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named : useData
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:262)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:418)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1269)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m19">javapracticeone.PracticeMethods#usingDropDown</h3><table class="result"></table><p class="totop"><a href="#summary">back to summary</a></p></body></html>